docs/ExpectAPI.md

```diff
@@ -646,6 +647,45 @@ test('the house has my desired features', () => {
 });
 ```
 
+### `toHaveProperty(propPath, value)`
```

Comment:
i checked lodash, and they use path.
chai uses `name`, but the also call it dot and braket notation.
at the same time `immutable` uses `keyPath` for `setIn` method :)

should we go with `keyPath` for consistency across fb projects? :)